.github/workflows/chromatic.yml

```diff
@@ -23,8 +23,6 @@ jobs:
           cache: "yarn"
       - name: Install dependencies
         run: yarn
-      - name: Build ag grid theme
```

Comment:
Why remove?

Reply:
no need, as we're not using sass anymore